src/compile_torture_tests.py

```diff
@@ -21,24 +21,6 @@
 
 import testing
 
-# TODO: Pass this in from build.py
-INSTALL_DIR = os.path.join(os.getcwd(),
```

Comment:
Probably yeah. It kind of accidentally evolved into the current form because I was taking the TODO too literally.
In theory it might be desirable to decouple the location of the sysroot path from this module, but 1) this still doesn't do that, and 2) there's no real pressure to do so right now.

Although, using `__file__` here means we add a relative path dependency to compile_torture_tests.py, so if we move this file things can break in non-obvious ways. Having considered that, I'd rather the dependency on file structure flow from build.py into this one, rather than them both re-deriving it, if that makes sense.